cmd/plugins/juju-metadata/validateagentsmetadata.go

```diff
@@ -104,6 +104,10 @@ Example bash snippet:
 juju metadata validate-agent-binaries -p ec2 -r us-east-1 -t ubuntu --juju-version 1.12.0 -d <some directory>
 RETVAL=$?
 [ $RETVAL -eq 0 ] && echo Success || echo "Failure"
+
+See also:
```

Comment:
Add a new line (so it'll render correctly on Discourse).

Reply:
I'm not sure where to add the new line, please advise.